pack/trors.json

```diff
@@ -651,339 +651,231 @@
 		"faction_code": "campaign",
-		"health_per_hero": false,
-		"hidden": false,
-		"is_unique": false,
 		"name": "Basic Recovery Upgrade",
 		"pack_code": "trors",
 		"pack_name": "The Rise of Red Skull",
 		"position": 162,
 		"quantity": 1,
 		"set_code": "hydra_camp",
-		"text": "Permanent. Setup.<br>\nYou get +4 hit points.<br>\nYour alter-ego gets +1 REC.<br>\n",
-		"threat_fixed": false,
+		"text": "Permanent. Setup.\nYou get +4 hit points.\nYour alter-ego gets +1 REC.",
 		"traits": "Condition.",
 		"type_code": "upgrade"
 	},
 	{
-		"base_threat_fixed": false,
 		"code": "04162b",
 		"deck_limit": 1,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "campaign",
-		"health_per_hero": false,
-		"hidden": true,
-		"is_unique": false,
 		"name": "Improved Recovery Upgrade",
+    "hidden": true,
 		"pack_code": "trors",
 		"position": 162,
 		"quantity": 1,
 		"set_code": "hydra_camp",
-		"text": "Permanent. Setup.<br>\nYou get +4 hit points.<br>\nYour alter-ego gets +1 REC.<br>\n<b>Response:</b> After you use your REC, exhaust this card → draw 1 card.",
-		"threat_fixed": false,
+		"text": "Permanent. Setup.\nYou get +4 hit points.\nYour alter-ego gets +1 REC.\n<b>Response:</b> After you use your REC, exhaust this card → draw 1 card.",
 		"traits": "Condition.",
 		"type_code": "upgrade"
 	},
 	{
-		"base_threat_fixed": false,
 		"code": "04163",
 		"deck_limit": 1,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "campaign",
 		"flavor": "\"A beautiful parasite growing inside the world's most secure computer systems.\" -- Arnim Zola",
-		"health_per_hero": false,
-		"hidden": false,
 		"illustrator": "Andrea Di Vitto & Laura Villari",
-		"is_unique": false,
 		"name": "Zola's Algorithm",
 		"pack_code": "trors",
 		"position": 163,
 		"quantity": 4,
 		"scheme_acceleration": 1,
 		"set_code": "expcamp",
 		"text": "<b>Alter-Ego Action</b> Exhaust your alter-ego and spend a [mental] resource → discard this card.",
-		"threat_fixed": false,
 		"type_code": "obligation"
 	},
 	{
-		"base_threat_fixed": false,
 		"code": "04164",
 		"deck_limit": 1,
-		"double_sided": false,
-		"escalation_threat_fixed": false,
 		"faction_code": "campaign",
 		"flavor": "\"This is going to need sutures.\" -- Stephen Strange",
-		"health_per_hero": false,
-		"hidden": false,
-		"is_unique": false,
 		"name": "Medical Emergency",
 		"pack_code": "trors",
 		"position": 164,
 		"quantity": 4,
 		"set_code": "expcamp",
-		"text": "<b>Forced Response:</b> At the end of your turn, take 1 damage if you are in hero form.<br>\n<b>Alter-Ego Action:</b> Discard the top 5 cards of your deck and spend a [physical] resource → discard this card.",
-		"threat_fixed": false,
+		"text": "<b>Forced Response:</b> At the end of your turn, take 1 damage if you are in hero form.\n<b>Alter-Ego Action:</b> Discard the top 5 cards of your deck and spend a [physical] resource → discard this card.",
```

Comment:
Are the campaign obligations expected to also be _hidden_ ?

Reply:
I think we would want to make them `hidden` if we didn't want them to show in the card list. I think we want them to show them, unless I'm missing something. 😅

Reply:
Oh ok, but maybe they should be a `spoiler`? 👍

Reply:
Sorry, I meant `spoiler` not `hidden` 🤦 

I feel like all the cards showing in the card list as being from the `Campaign` class should probably be tagged as a `spoiler`, but happy to check what's the general consensus on it.